taqueria-plugin-taquito/instantiate_account.ts

```diff
@@ -0,0 +1,48 @@
+import { getCurrentEnvironmentConfig, sendAsyncErr, sendJsonRes, sendWarn } from '@taqueria/node-sdk';
+import {
+	generateAccountKeys,
+	getDeclaredAccounts,
+	getEnvTypeAndNodeConfig,
+	getNetworkInstantiatedAccounts,
+	InstantiateAccountOpts as Opts,
+} from './common';
+
+const instantiate_account = async (parsedArgs: Opts): Promise<void> => {
+	const env = getCurrentEnvironmentConfig(parsedArgs);
+	if (!env) return sendAsyncErr(`There is no environment called ${parsedArgs.env} in your config.json`);
+	try {
+		const [envType, nodeConfig] = await getEnvTypeAndNodeConfig(parsedArgs, env);
+		if (envType !== 'Network') {
+			return sendAsyncErr('taq instantiate-account can only be executed in a network environment');
+		}
+
+		const declaredAccountAliases = Object.keys(getDeclaredAccounts(parsedArgs));
+		const instantiatedAccounts = getNetworkInstantiatedAccounts(nodeConfig);
+
+		let accountsInstantiated = [];
+		for (const declaredAccountAlias of declaredAccountAliases) {
+			if (!instantiatedAccounts.hasOwnProperty(declaredAccountAlias)) {
+				await generateAccountKeys(parsedArgs, nodeConfig, declaredAccountAlias);
+				accountsInstantiated.push(declaredAccountAlias);
+			} else {
+				sendWarn(
+					`Note: ${declaredAccountAlias} is already instantiated in the current environment, "${parsedArgs.env}"`,
+				);
+			}
+		}
+
+		if (accountsInstantiated.length !== 0) {
+			return sendJsonRes(
+				`Accounts instantiated: ${
+					accountsInstantiated.join(', ')
+				}.\nPlease execute "taq fund" targeting the same environment to fund these accounts`,
+			);
+		} else {
+			return sendJsonRes(`No accounts were instantiated`);
+		}
+	} catch {
+		return sendAsyncErr('No operations performed');
```

Comment:
And in here, include the original error?

Reply:
What do you mean?

Reply:
The errors before this error (`'No operations performed'`) will tell the user if it came from an `originate` task or `transfer` task if that's what you meant. This is more like to assure users that no operations were injected into the chain.

Reply:
I mean, here we catch any possible error, but do not include any information from them in the error message, so the user knows that "No operations performed" but not why.

Reply:
Oh it's caught in the `handleOpsError` function